CanvasPlusPlayground/Features/Assignments/AssignmentDetailView.swift

```diff
@@ -0,0 +1,55 @@
+//
+//  AssignmentDetailView.swift
+//  CanvasPlusPlayground
+//
+//  Created by Ethan Fox on 2/7/25.
+//
+
+import SwiftUI
+
+struct AssignmentDetailView: View {
+    let assignment: AssignmentAPI
+    init(assignment: AssignmentAPI) {
+        self.assignment = assignment
+    }
+    var body: some View {
+        if (assignment.submissionTypes.contains("online_quiz")) {
+            WebView(url: URL(string:assignment.html_url ?? "google.com")!)
+        } else {
+            Form {
+                Section {
+                    HStack {
+                        Text("Assignment Name")
+                        Spacer()
+                        Text(assignment.name)
+                    }
```

Comment:
You can use `LabeledContent` throughout here (remembered this existed recently!)
Also maybe use "Name" instead of "Assignment Name" since it becomes 2 lines
```suggestion
                    LabeledContent("Name", value: assignment.name)
```

Reply:
👍 [d4a7f7a](https://github.com/gtiosclub/CanvasPlusPlayground/pull/208/commits/d4a7f7afa0328bc8f9505a8e5345e79fd6d5bcdc)